src/snowflake/snowpark/functions.py

```diff
@@ -10722,6 +10723,63 @@ def sproc(
         )
 
 
+def _call_model(
+    model_name: str,
+    version_or_alias_name: str | None,
+    method_name: str,
+    *args,
+    _emit_ast: bool = True,
+) -> Column:
+    if _emit_ast:
+        _ast = build_function_expr(
+            "model", [model_name, version_or_alias_name, method_name, *args]
+        )
+    else:
+        _ast = None
+
+    args_list = parse_positional_args_to_list(*args)
+    expressions = [Column._to_expr(arg) for arg in args_list]
+    return Column(
+        ModelExpression(
+            model_name,
+            version_or_alias_name,
+            method_name,
+            expressions,
+        ),
+        _ast=_ast,
+        _emit_ast=_emit_ast,
+    )
+
+
+@publicapi
+def model(
+    model_name: str,
+    version_or_alias_name: str | None = None,
+    _emit_ast: bool = True,
+) -> Callable:
```

Comment:
is there an official doc that introduce these models?
I want to see how these models are used natively by sql users